### DISCUSSION TITLE
2/7

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#63] [Announcements]
# 2/7

SICP Python 中文翻译

https://composingprograms.netlify.app/2/7

**Comment by T1ing1** [comment]
感谢有你

**Comment by Alan-Musk** [comment]
感谢大佬

**Comment by zhangeecs** [comment]
小错误 ：< 2.7.1 通过在用户定义类种实现这个相同的方法，>
“种” --> “中”

**Reply by zhangeecs** [reply]
<2.7.3 代码片段 ComplexMA(1, 1 *>
应为：ComplexMA(1, 1 * pi)

**Reply by mancuoj** [reply]
感谢，已修改~

**Comment by marvoalou** [comment]
泛型函数line 13：’来正确的处所有的数值类型‘  少了‘处理’的‘理’

**Reply by mancuoj** [reply]
感谢，已修改~
